.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+- [] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
+- [] **New feature**: non-breaking change that adds functionality.
+- [] **Breaking change**: fix or feature that will cause current functionality to change.
 
 ## Process & policy checklist
 <!--- Review the list and check the boxes that apply. -->
-- [ ] I have added tests for my changes.
-- [ ] I have tested my changes.
-- [ ] I have updated the project documentation to reflect my changes.
-- [ ] I have read the CONTRIBUTING documentation and followed the standards set for this project.
 
- 
-Click here for details on [formatting pull requests](https://microsoft.github.io/fast-dna/docs/contributing/working#formatting-messages).
+- [] I have added tests for my changes.
+- [] I have tested my changes.
+- [] I have updated the project documentation to reflect my changes.
+- [] I have read the CONTRIBUTING documentation and followed the standards set for this project.
+
+## Pull request formatting
```

Comment:
I recommend a few changes here: 

- Lets reference in a comment at the top that the directions are here below. Given the size of the window, the user won't see them until after they have done their title without some kind of note up top. I agree with not putting them at the top though since it would move the rest down too far.

- Lets put all directions in comment form so you don't see it all on the pull request page (once it is submitted).


-